nixos/tests/installer.nix

```
@@ -991,6 +991,68 @@ in {
     '';
   };
 
+  bcachefsLinuxTesting = makeInstallerTest "bcachefs-linux-testing" {
+    extraInstallerConfig = {
+      imports = [ no-zfs-module ];
+
+      boot = {
+        supportedFilesystems = [ "bcachefs" ];
+        kernelPackages = mkForce pkgs.linuxPackages_testing;
+      };
+    };
+
+    extraConfig = ''
+      boot.kernelPackages = lib.mkForce pkgs.linuxPackages_testing;
```

Since this is using `lib.mkDefault` above. Do we need force here?
```suggestion
      boot.kernelPackages = pkgs.linuxPackages_testing;
```

---

We shouldn't need to force it AFAIK.